visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.spec.ts

```diff
@@ -276,4 +292,45 @@ describe("MapTreeViewLevelController", () => {
 			expect(mapTreeViewLevelController["_viewModel"].collapsed).toBeFalsy()
 		})
 	})
+	describe("getNodeUnary", () => {
+		it("should return the unary of the current node", () => {
+			mapTreeViewLevelController["node"] = VALID_NODE_WITH_METRICS
+
+			const nodeUnary: number = mapTreeViewLevelController.getNodeUnary()
+
+			expect(nodeUnary).toBe(VALID_NODE_WITH_METRICS.attributes["unary"])
+		})
+	})
+	describe("getUnaryPercentage", () => {
+		it("should return the Child Node Unary-Percentage to 50 percent", () => {
+			mapTreeViewLevelController["node"] = VALID_NODE_WITH_ROOT_UNARY.children[0]
+
+			const nodePercentage = mapTreeViewLevelController.getUnaryPercentage() //leave for later
```

Comment:
comment?

Reply:
comments can be removed, there used to be no expectation, so I felt the need to clarify the const

Reply:
also try to use names like `result` or `actual` for whatever you receive from your method you're currently testing.
This way you will get statements like `expect(actual).toBe(expected)` or `expect(result).toBe(expected)`. You don't have to move smaller statements into their own variables though, such as:
```
const expected = "foo"

const result = myMethod()

expect(result).toBe(expected)
```
instead do
```
const result = myMethod()

expect(result).toBe("foo")
```

Reply:
Noted :)